### ISSUE TITLE
dns_ispconfig.sh broken

### ISSUE THREAD

**Issue by remkohat** [acmesh-official/acme.sh#3895]
Latest alterations in **dns_ispconfig.sh** broke the script!
As a result acme.sh is saying "You haven't specified the ISPConfig Login data" though it is specified in account.conf.
This happens when running the cron to autorenew and also when trying to get a new certificate from the command line.

After I replaced the script with the previous version all is working fine again.

**Comment by Neilpang** [comment]
3.0.2 is not released yet.  3.0.1 is the latest released version.

**Comment by remkohat** [comment]
Released or not, the autoupdate function in acme.sh apparently updates it to 3.0.2
And with a broken dns_ispconfig.sh

**Comment by andzio** [comment]
/root/.acme.sh/acme.sh -v
https://github.com/acmesh-official/acme.sh
v3.0.2

/root/.acme.sh/acme.sh --server https://acme-v02.api.letsencrypt.org/directory --issue --dns dns_ispconfig --keypath /etc/pve/local/pveproxy-ssl.key --fullchainpath /etc/pve/local/pveproxy-ssl.pem --reloadcmd "systemctl restart pveproxy" -d domain.com

You haven't specified the ISPConfig Login data, URL and whether you want check the ISPC SSL cert. Please try again.

**Comment by remkohat** [comment]
@Neilpang In my case different paths but exactly the same result as @andzio 

@andzio As a workaround for now overwrite dns_ispconfig.sh with the previous version of the file (Jan 16 2021)

**Comment by andzio** [comment]
Yes, the old version works ok, so I overwritten it, but it's just a denture.

**Comment by Neilpang** [comment]
fixed,   can you guys please try with the latest dev branch?

```
acme.sh --upgrade -b dev
```

**Comment by iNetSpec** [comment]
@Neilpang   I essentially already tested by editing the file on two of my machines that were giving the same error as above before reporting the commit as the problem; so "works for me".

**Comment by RBCOMGmbH** [comment]
> fixed, can you guys please try with the latest dev branch?
> 
> ```
> acme.sh --upgrade -b dev
> ```

Without manually changing anything I can also confirm that your changes in the dev branch worked. 
Thank you for fixing it!
